gradle-sls-packaging/src/main/resources/init.sh

```diff
@@ -60,9 +60,8 @@ start)
     fi
     printf "%-50s" "Running '$SERVICE'..."
 
-    # ensure log and pid directories exist
-    mkdir -p "var/log"
-    mkdir -p "var/run"
+    # ensure log, pid and tmp directories exist
+    mkdir -p "var/log" "var/run" "var/data/tmp"
```

Comment:
can likely remove the distTar steps. The issue with depending on distTar is that it can only make this directory during install, but we need this directory exist for every application run.